.travis.yml

```diff
@@ -82,6 +82,12 @@ jobs:
         - yosys -V
         - "travis_wait 30 sleep 1800 &"
         - ./.run_formal_checks.sh Ops
+    - stage: test
+      name: "Formal equivalence: AddNot"
+      script:
+        - yosys -V
+        - "travis_wait 30 sleep 1800 &"
+        - ./.run_formal_checks.sh AddNot
```

Comment:
Thoughts on putting this as a test in the ScalaTests rather than a full Travis stage?

Reply:
The DUT and reference both use the `VerilogEmitter` from the current version of the package, so how could we find emitter bugs with that?

Qui Verilogiet ipsos Veriloges?

Reply:
Good point about the wastefulness of the stage.

I put it into the same stage as `Ops` since they both run fast. I think this will cache the jars for your script?

This will also help cut Travis-related queueing delays.

Reply:
Ops is definitely better but I was thinking the style that `CombineCats` uses https://github.com/freechipsproject/firrtl/blob/b90589f5cd9d4048ada2a05d5225874791546170/src/test/scala/firrtlTests/transforms/CombineCatsSpec.scala#L34 where it checks against the `MinimumVerilogCompiler` that excludes this stuff

Reply:
`MinimumVerilogEmitter` still includes `InlineNots`. The only thing it filters is `DeadCodeElimination`.

Reply:
I think that fixing `MinimumVerilogEmitter` to be a more meaningful "safe baseline" would be good, but for this PR my tinfoil hat is firmly attached :)

Reply:
> MinimumVerilogEmitter still includes InlineNots. The only thing it filters is DeadCodeElimination

Ah well that's not good. We should make it truly Minimum. Anyway for this PR I think what you're doing is right then.